packages/amplication-data-service-generator/src/server/auth/user-info/create-user-info.ts

```diff
@@ -1,75 +0,0 @@
-import { Module } from "../../..";
-import DsgContext from "../../../dsg-context";
-import {
-  CreateUserInfoParams,
-  EventNames,
-  types,
-} from "@amplication/code-gen-types";
-import { readFile } from "../../../util/module";
-import { interpolate, removeTSClassDeclares } from "../../../util/ast";
-import { builders, namedTypes } from "ast-types";
-import { print } from "recast";
-import { getUserIdType } from "../../../util/get-user-id-type";
-import pluginWrapper from "../../../plugin-wrapper";
-
-const templatePath = require.resolve("./user-info.template.ts");
-
-export async function createUserInfo(): Promise<Module[]> {
-  const { serverDirectories } = DsgContext.getInstance;
-  const authDir = `${serverDirectories.srcDirectory}/auth`;
-  const template = await readFile(templatePath);
-  const templateMapping = prepareTemplateMapping();
-  const filePath = `${authDir}/UserInfo.ts`;
-  return pluginWrapper(createUserInfoInternal, EventNames.CreateUserInfo, {
-    template,
-    templateMapping,
-    filePath,
-  });
```

Comment:
we are removing here the plugin event  - please make sure this was intended. also, we need to update the docs

Reply:
@yuval-hazaz I think we should delete all the auth folder because this logic will be implemented in the plugin, so the is no need for any of the current plugin events.